scripts/scripts.py

```diff
@@ -0,0 +1,38 @@
+# ruff: noqa: S602, S607
+import subprocess
+
+
+def test():
+    """Run pytest tests."""
+    subprocess.run("pytest tests", shell=True, check=True)
```

Comment:
[suggestion/question] Will this run via UV or directly as Python? This means we would need to have all the packages available on the path for the users, which might be tricky or unexpected. We should ensure that we use UV to run those scripts.

Reply:
Maybe this should be made more clear in the readme.